pkg/datastore/logger.go

```diff
@@ -37,6 +37,13 @@ const (
 	SAAS_PERSISTENCE = "persistence"
 )
 
+var TRACE = func(format string, v ...any) {
+}
```

Comment:
Why is this needed? Why not just have `log.Trace()` function calls, which will take effect if the log level is set to _Trace_?

Reply:
TRACE can be used anywhere without the need to create a logger instance to be passed around and this is just to disable logging when TRACE is not enabled. check TRACE initialization below.